### ISSUE TITLE
add support for LZBITMAP

### ISSUE THREAD

**Issue by blacktop** [lzfse/lzfse#62]
Will you please open source the LZBITMAP algo as well?

**Comment by jevinskie** [comment]
Yes, please, it is unsavory to have to rely on closed-source Apple libraries.

**Comment by IceCooler148** [comment]
This algorithm is now used a lot by apple in the new IPSW versions, and become more and more in use by them.

Please open source this algo 🙏

**Comment by eafer** [comment]
Hi @blacktop @IceCooler148 @jevinskie @cgohlke @cxnder, I have a library for LZBITMAP: https://github.com/eafer/libzbitmap. Much slower than the original for now, but at least it works.

**Comment by blacktop** [comment]
DAAAANG you Corellium guys are rrreally good at blackbox RE! 😎 👍
